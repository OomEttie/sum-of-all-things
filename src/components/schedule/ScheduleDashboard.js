import React from 'react';
import { Link } from 'react-router-dom';
import { connect } from 'react-redux';

import ScheduleList from './ScheduleList';

export class ScheduleDashboard extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div className="content-container">
        <ScheduleList schedules={this.props.schedules} />
      </div>
    );
  }
}

const mapStateToProps = (state, props) => {
  return {
    schedules: state.schedules
  };
};

const mapDispatchToProps = dispatch => {};

export default connect(mapStateToProps)(ScheduleDashboard);
