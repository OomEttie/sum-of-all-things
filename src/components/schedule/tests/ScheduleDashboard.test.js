import React from 'react';
import { shallow } from 'enzyme';
import { ScheduleDashboard } from '../ScheduleDashboard';

test('should render ScheduleDashboard correctly', () => {
  const wrapper = shallow(<ScheduleDashboard />);
  expect(wrapper).toMatchSnapshot();
});

