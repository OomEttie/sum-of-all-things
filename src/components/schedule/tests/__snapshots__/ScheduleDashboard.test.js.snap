// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ScheduleDashboard correctly 1`] = `
<div
  className="content-container"
>
  <ScheduleList
    schedules={Array []}
  />
</div>
`;
